spec/menu_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+require 'menu'
+
+describe Menu do
+  subject(:menu) { Menu.new }
+  let(:items) { { :'Pizza' => 9.25, :'Chips' => 2.00 } }
```

Comment:
Actually, in production they wouldn't be symbols at all - I should probably just use strings here.